var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -16,6 +16,7 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
 
     license("BSD-3-Clause")
 
+    version("0.4.1", sha256="ba95f26475ad68da1f3a24d091dc1b925525e269e4c83c1eaf1d37d29b526666")
     version("0.4.0", sha256="34fd0da0d1a72b6981bed0bba029ba0947e0d0d99beb3e0aad0a478095c9527d")
```

Comment:
We may want to consider setting the older versions as deprecated, but I don't know if it makes a big difference? What do you think?

Reply:
Nut sure.... maybe @albestro has more clues...

Reply:
It's a good point.

IMHO we have already implicitly deprecated, i.e. we would suggest to use the latest because there is no point in using an old one, most of the past `0.*` releases without actually marking them, isn't it?! I would not care too much for the moment.